src/queries/index.js

```diff
@@ -47,6 +53,7 @@ export const getQuery = (query, filterOptions = {}) => {
 export const getFetchMoreQuery = (query, filterOptions = {}) => {
   const FETCH_MORE_QUERIES = {
     [QUERY_TYPES.EVENT_RECORDS]: GET_EVENT_RECORDS,
+    // TODO: check if GET_NEWS_ITEMS would be enough here, as the query gets a dataProvider param or not, but the rest is the same (like for event records)
     [QUERY_TYPES.NEWS_ITEMS]: filterOptions.showNewsFilter
       ? GET_FILTERED_NEWS_ITEMS
       : GET_NEWS_ITEMS
```

Comment:
getFetchMoreQuery I thought is for "scroll up to refresh" .. but it seams is not for that. So why are you fetching more data ?

Reply:
"pull to refresh" is when you are on top of the list and pull the screen down. then the query for refresh is triggered.
"fetch more" is used when you scroll down through the list. always on reaching the end of the list more data will be fetched, i think 15 in our case. the number is set in the query variables.